cpp/stdlib.cc

```diff
@@ -139,10 +240,10 @@ time_t time() {
 
 // NOTE(Jesse): time_t is specified to be an arithmetic type by C++. On most
 // systems it's a 64-bit integer.  64 bits is used because 32 will overflow in
-// 2038.  Someone on a comittee somewhere thought of that when moving to 64-bit
-// architectures to prevent breaking ABI again; on 32-bit systems it's usually
-// 32 bits.  Point being, using anything but the time_t typedef here could
-// (unlikely, but possible) produce weird behavior.
+// 2038.  Someone on a comittee somewhere thought of that when moving to
+// 64-bit architectures to prevent breaking ABI again; on 32-bit systems it's
+// usually 32 bits.  Point being, using anything but the time_t typedef here
+// could (unlikely, but possible) produce weird behavior.
```

Comment:
Huh. I've noticed `clang-format` can be a little non-deterministic at times... (either that or this wasn't formatted before...)